src/components/common/Tile/PersonTile/styled.jsx

```diff
@@ -1,83 +1,59 @@
 import styled from "styled-components";
 import { ReactComponent as person } from "../../../../assets/icons/person.svg";
 
-
 export const Wrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-start;
-  align-content: center;
-  width: 208px;
-  height: auto;
-  padding: 16px;
-  box-shadow: ${({ theme }) => theme.properties.tile.boxShadow};
-  gap: 12px;
-
-  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.s}) {
-    justify-self: center;
-    width: 136px;
-    max-height: 300px;
-    gap: 8px;
-  }
+	display: grid;
+	grid-template-rows: auto 1fr;
+	padding: 16px;
+	gap: 12px;
+	height: 100%;
+	box-shadow: ${({ theme }) => theme.properties.tile.boxShadow};
 `;
 
 export const PersonIcon = styled(person)`
-  width: 72px;
-  height: 72px;
-  
+	width: 72px;
+	height: 72px;
 `;
-export const ImageWrapper = styled.div`
-  display: flex;
-  border-radius: 5px;
-  flex-direction: column;
-  align-items: center;
-  width: 100%;
-  background-color: #C4C4C4;
-  height: 250px;
-  justify-content: center;
 
-  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.s}) {
-    width: 114px;
-    min-width: 114px;
-    height: 169px;
-    min-height: 169px;
-  }
+export const ImageWrapper = styled.div`
+	border-radius: 5px;
+	background-color: ${({ theme }) => theme.colors.tile.imageWrapper};
+	aspect-ratio: 6/9;
+	display: flex;
+	justify-content: center;
+	align-items: center;
 `;
```

Comment:
I think this change is causing problems
<img width="1034" alt="Zrzut ekranu 2023-11-25 o 02 17 09" src="https://github.com/jakub-bartoszek-developer/Movie-browser/assets/131645596/70d76cd9-6f50-40b5-a4cb-738c9fede260">